library/odoo

```diff
@@ -0,0 +1,4 @@
+# maintainer: Aaron Bohy <aab@odoo.com> (@aab-odoo)
+
+8.0: git://github.com/odoo/docker@2aae53084d32aaf44d3630e2a09d685bf971d81c 8.0
+8: git://github.com/odoo/docker@2aae53084d32aaf44d3630e2a09d685bf971d81c 8.0
```

Comment:
Actually, v8 is not the latest version. I put the tag `stable` instead. I also fixed the reference.